mopidy_gmusic/library.py

```diff
@@ -585,9 +585,9 @@ def _to_mopidy_album(self, song):
         artist = self._to_mopidy_album_artist(song)
         date = str(song.get("year", 0))
 
-        album_id = song.get("albumId")
-        if album_id is None:
-            album_id = create_id(artist.name + name + date)
+        #album_id = song.get("albumId")
+        #if album_id is None:
+        album_id = create_id(artist.name + name + date)
```

Comment:
Assuming that this change is good to go and works with All Access too (I can't confirm, as I don't have a subscription and don't use GMusic), the commented out code should be removed before this is merged.

Reply:
This allows for conflicts if the artist, album name, and date are the same. Unlikely, but still possible. One of the reasons I had for sticking to pulling IDs like this from Google if they exist were specifically because of this. One specific example would be for two Various Artists albums with the same name in the same year.

Reply:
Yeah, I get that there's possibility of conflict. For me though, with only uploaded tracks, this change is way better, and makes the extension work really well, rather than having to wade through all the duplicates. 
I don't know how well it works for subscription. I know that, with Spotify, they sometimes have the same album on there multiple times. If gmusic does that, then that could potentially cause problems.

Reply:
FYI Weezer released two self-titled albums last year. 🙄

Reply:
Other than that, I think it's extremely unlikely that someone will have uploaded two conflicting albums. And, if they have, this change is still an overall improvement, in my opinion.

Reply:
I'm not a huge fan of knowingly breaking things, but considering it's broken in other more obvious ways, I can get behind this.

My one request - can you use some sort of separator (like : or |) between the fields? I think it will make it a bit more stable and make the IDs easier to read if we're ever debugging.

Reply:
> can you use some sort of separator (like : or |) between the fields?

Sound like a good idea. I'll do that.

Reply:
I'll go with the `|` as I've got at least one album with `:` in the title.
```
Various Artists|Fat Music, Vol. 5: Live Fat Die Young|2001
```